adapters/handlers/rest/clusterapi/indices_payloads.go

```diff
@@ -223,6 +224,47 @@ func (p objectListPayload) Unmarshal(in []byte) ([]*storobj.Object, error) {
 	return out, nil
 }
 
```

Comment:
The handler for the new endpoint is missing in `hanlders/reset/clusterapi`

Reply:
done